src/utils/auth.js

```diff
@@ -42,8 +42,13 @@ export function getCurrentOrgLocal(username) {
 
 export function saveCurrentOrgLocal(username, org) {
   const key = CURRENT_ORG_KEY + ':' + username
-  localStorage.setItem(key, JSON.stringify(org))
-  VueCookie.set('X-JMS-ORG', org.id)
+  if (org) {
+    localStorage.setItem(key, JSON.stringify(org))
+    VueCookie.set('X-JMS-ORG', org.id)
+  } else {
+    localStorage.removeItem(key)
+    VueCookie.delete('X-JMS-ORG')
+  }
 }
 
 export function setPreOrgLocal(username, org) {
```

Comment:
I checked the given snippet and found no known issue. It seems fine, with some minor changes like adding curly braces around the "localStorage" lines to keep it consistent in coding style conventions.

Here's what I suggest:
1. No obvious issues were highlighted.
2. Ensure that all imports are correct as per version used or consider using a different import statement according to modern JavaScript practices. (Assuming this context is from an older version of Node.js when these statements might be more common.)
3. For better clarity on global functions and their usage patterns, add comments at appropriate points explaining their purpose, parameters, return value etc., if not already done (especially considering the current code).
4. If you plan for any future rework or modification to the core logic, make sure to update docstrings where necessary to clearly state how they should interact with other parts of the application or system. Always maintain compatibility across different versions.